src/topology/basic.lean

```diff
@@ -7,8 +7,6 @@ import order.filter.ultrafilter
 import order.filter.partial
 import data.support
 
-noncomputable theory
-
 /-!
```

Comment:
Seems it lints if there is _anything_ non-import above it, even though the docstring generator deals with it fine